orttraining/orttraining/test/python/orttraining_test_ortmodule_autograd.py

```diff
@@ -1210,3 +1209,63 @@ def forward(self, model_input):
     assert not can_run
 
     del os.environ["ORTMODULE_SKIPPED_AUTOGRAD_FUNCTIONS"]
+
+
+def test_pythonop_training_mode():
+    def check_pythonop_training_mode(model, is_eval_mode):
+        ## make sure the ort's PythonOp's training_mode is correct
+        if is_eval_mode:
+            onnx_nodes = (
+                model._torch_module._execution_manager._inference_manager._onnx_models.exported_model.graph.node
```

Comment:
nit: shall we disable the pylint W0212. You can check the beginning of this file for how to disable using error num.